modules/javafx.graphics/src/main/java/javafx/scene/layout/CornerRadii.java

```diff
@@ -312,18 +312,20 @@ public CornerRadii(
                 topRightVerticalRadiusAsPercent || topRightHorizontalRadiusAsPercent ||
                 bottomRightHorizontalRadiusAsPercent || bottomRightVerticalRadiusAsPercent ||
                 bottomLeftVerticalRadiusAsPercent || bottomLeftHorizontalRadiusAsPercent;
-        uniform = topLeftHorizontalRadius == topRightHorizontalRadius &&
-                topLeftVerticalRadius == topRightVerticalRadius &&
+        uniform = topLeftHorizontalRadius == topLeftVerticalRadius &&
```

Comment:
I found the diff a bit troubling to read, but from what I gather, all the radii need to equal to qualify for uniformity, and what you did here is add a `topLeftHorizontalRadius == topLeftVerticalRadius` (and percent version) to this check.

You also already improved the readability by using `topLeftHorizontalRadius` everywhere.  It might be even better to extract this one to a new variable, so it is even more clear that you're comparing all the other radii with the same value.

Reply:
I tried that, but in my eyes it didn't clearly improve readability beyond the current version.